src/cycax/cycad/assembly_blender.py

```diff
@@ -105,8 +108,9 @@ def colour(self, colour: str, part: str) -> str:
         """
         working_part = f"{part}_{self.parts[part]}"
         template_object = bpy.data.objects.get(working_part)
+        colour_rgb = colour_to_rgb[colour.upper()]
         matcolour = bpy.data.materials.new(colour)
-        matcolour.diffuse_color = (0, 1, 0, 0.8)
+        matcolour.diffuse_color = (colour_rgb[0] / 255, colour_rgb[1] / 255, colour_rgb[2] / 255, 0.8)
```

Comment:
FYI: it might be better to use a colour library from somewhere else: e.g. matplotlib.

```
In [11]: import matplotlib.colors as mcolors
In [12]: mcolors.to_rgb("BLUE")
Out[12]: (0.0, 0.0, 1.0)
In [13]: mcolors.to_rgb("Blue")
Out[13]: (0.0, 0.0, 1.0)
In [14]: mcolors.to_rgb("blue")
Out[14]: (0.0, 0.0, 1.0)
```

It will be simpler to point users to the matplotlib colour pallet: https://matplotlib.org/stable/gallery/color/named_colors.html